libethashseal/genesis/test/ByzantiumToConstantinopleFixAt5Test.cpp

```diff
@@ -26,7 +26,7 @@ static std::string const c_genesisInfoByzantiumToConstantinopleFixAt5Test = std:
 		"EIP150ForkBlock": "0x00",
 		"EIP158ForkBlock": "0x00",
 		"byzantiumForkBlock": "0x00",
-		"constantinopleForkBlock": "0x00",
+		"constantinopleForkBlock": "0x05",
```

Comment:
Please explain in the commit message what is changed and why.

Reply:
because Constantinople and ConstantinopleFix activate at the same time on mainnet

Reply:
> in the commit message